Interview/demo.md

```diff
@@ -22,62 +22,74 @@ Before proceeding to the section of your choice, please consider the following t
+* We left out of the requirements whether or not you should try to download everything (photos + each photo detail). This is up to you to decide and to justify.
 * Testing is very important for us. Even if you don't write a single test (for instance, because of time constraints), your app should be easy to test (we love [Dependency injection](https://en.wikipedia.org/wiki/Dependency_injection)).
 * Error scenarios should be taken into consideration and it should be easy to add them, even if you don't explicitly handle them (e.g. using an `UIAlertController`).
 * Although UI and UX are important, we are more concerned in this demo with your thought process and with how you architect your application. Your demo should take into consideration features that might be added in the future.
-* You can use any 3rd party libraries you wish (`Alamofire`, `ReactiveCocoa`, `PromiseKit`, `Realm`, etc) but be prepared to justify why you did so. Feel free to use package managers to handle them.
+* You can use any 3rd party libraries (except for networking) if you wish ( `ReactiveCocoa`, `PromiseKit`, `Realm`, etc) but be prepared to justify why you did so. Feel free to use package managers to handle them.
 * **Be consistent in your code**. We advise using something like [raywenderlich's swift style guide](https://github.com/raywenderlich/swift-style-guide) while doing the demo. It's absolutly fine to use any other style, as long as you are consistent.
 * Clean the file project structure and remove any unused methods (e.g., from `AppDelegate`). This shows attention to detail.
 * Be opinionated regarding any architecture you use and take your time to make it a reflection of your thought process.
 * We don't have a submission deadline so take your time to polish your project.
 
 ### 1. The babylon demo project
 
-From a high level point of view the demo consists of a list of posts, where each post has its own detail.
+From a high level point of view the demo consists of a list of photos. Upon tapping any photo, the user is taken to the photo detail screen.
 
-#### Posts Screen
+#### Photos List Screen
 
-A post has a title and it's up to you how to display it. To retrieve the posts, you can use the following API:
+A Photo has a thumbnail image and a title and you can decide how best to display it. To retrieve the photos, you can use the following API:
 
-* http://jsonplaceholder.typicode.com/posts
+* http://jsonplaceholder.typicode.com/photos
 
-When a post is tapped, you should go to the detail screen.
+When a photo is tapped, you should go to the Photo detail screen.
+Also, a photo that is marked favourite shows up first in the list followed by other photos.
 
-#### Detail screen
+#### Photo Detail screen
 
-A post detail screen will have:
+A Photo detail screen will have:
 
-* Its author.
-* Its description (via the `body`).
-* The number of comments it has.
+* The user the photo belongs to.
```

Comment:
I feel like this one is a breeze of fresh air and may bring new technical challenges, 💯 about changing it 👍